deploy/patch-yaml.sh

```diff
@@ -18,7 +18,7 @@ if [ "$(uname)" = "Darwin" ]; then
 fi
 
 DIR=`dirname $0`
-find $DIR -name development -exec rm -r {} \;
+rm -rf ./operator/overlays/development ./overlays/development
```

Comment:
The scripts need to use $DIR as the base dir, otherwise they will not work correctly when executed from another directory.